### ISSUE TITLE
Slight consideration to choosing marked bicycle routes

### ISSUE THREAD

**Issue by Asteliks** [abrensch/brouter#443]
As I understand brouter has been set to prefer marked bicycle routes which is a good idea in general but I would like to suggest a minor exception to that rule. Consider the famous [EuroVelo 10]( https://en.eurovelo.com/ev10/southern-coast). Some parts of this route, in Poland at least, are truly unpassable due to crossing a [literal dessert of full of sand]( https://www.openstreetmap.org/note/3232599). My suggestion is that we add a minor override when the smoothness is set to something like impassable or horrible the standard routing profile shouldn’t priorities this route.

**Comment by poutnikl** [comment]
As an alternative, you may want to check some 3rd party user profiles, that do not automatically consider cycleroutes as perfect.

**Comment by Asteliks** [comment]
I know, personalny I have my very own profile but the default ones should give good routes for the majority of the users out there

**Comment by poutnikl** [comment]
I think it follows a particular easy approach of route prioritization that works for most users and cases. Routes seldom lead across very difficult or not passable terrain.

**Comment by Asteliks** [comment]
> I think it follows a particular easy approach of route prioritization that works for most users and cases. Routes seldom lead across very difficult or not passable terrain.
perhaps this is the case for Western Europe but here in Poland routes are often not maintained and the proposed exception would just deal with the very difficult and not passable terrain

> Also, there will be frequent cases of following the route having priority over avoiding difficult surfaces (within reason).
As mentioned by you within reason. A not passable route doesn’t seem to be within reason

**Comment by polyscias** [comment]
> Perhaps this is the case for Western Europe but here in Poland 

Yes, I am quite sure that holds for most of Europe so if something is needed it looks to for a small sub-set of users.

Had a look at the note and there is the remark that the R10 here seem to be incorrectly mapped, is that maybe more often the case when the route is bad?

**Comment by Asteliks** [comment]
I did some research on the R10 yesterday and based on several biker blogs it would seem that the mapping is correct. The R10 goes through a desert, loose sands and a swamp with missing bridges. 

I also had the pleasure of checking some marked trails in Warsaw with similar results. Based on my experience so far these marked traits seem to be underfunded or not for typical trekking bikes. A mountain bike with wide tiers should do without a problem

**Comment by poutnikl** [comment]
It is similar issue as it is/was for car profiles, considering large unpaved road network in many underdeveloped or large regions. What is good for implicit standards of paved tertiaries and up for Europe, it is less applicable for Eastern Africa, Canada or Siberia.

**Comment by Asteliks** [comment]
Well first of all we have a key called smoothness in OSM and the way it should be interpreted is whether a certain vehicle type can travers the track. Our default profile is for trekking bikes thus we should omit tracks that are very_bad, horrible, very_horrible and impassable it is all in our [wiki](https://wiki.openstreetmap.org/wiki/Key:smoothness). This defines how data should be mapped and how it should be interpreted.

Our approach to mapping works for developed countries as well as underdeveloped. What is more most of the mapping is done by local standards so a street considered bad in Germany might be mapped as intermediate on poorer countries thus everything still works.

**Comment by hungerburg** [comment]
Making _smoothness=impassable_ count like _highway=steps_ should be in the default. poutnikl's term _haulpenalty_ is quite on target.
